### PR TITLE
Adding refresh endpoint.

### PR DESCRIPTION
**Pull request by iulianp26**

### CODE REVIEW

File: app.go [hunk 1]
```diff
@@ -35,7 +36,7 @@ func main() {
 		Name:   "cms-metadata-notifier-address",
 		Value:  "",
 		Desc:   "Address of the cmsMetadataNotifier",
-		EnvVar: "CMS_METADATA_NOTIFIER_ADDR",
+		EnvVar: "CMS_MEqTADATA_NOTIFIER_ADDR",
```
Comment (tosan88) [inline app.go]: Typo

File: app.go [hunk 2]
```diff
@@ -89,19 +94,20 @@ func main() {
...
+	mm.mappings = fetchMappings(mm.config.mappingURL)
+	infoLogger.Printf("%v", mm.prettyPrintMappings())
 }
 
 func listen(mm metadataMapper, hc healthcheck) {
 	r := mux.NewRouter()
 	r.HandleFunc("/notify", mm.handleNotification).Methods("POST")
 	r.HandleFunc("/__health", hc.health()).Methods("GET")
 	r.HandleFunc("/__gtg", hc.gtg).Methods("GET")
+	r.HandleFunc("/refresh", mm.handleRefresh).Methods("POST")
```
Comment (tosan88) [inline app.go]: Can we have a `__reload` endpoint instead the naming to match with other apps?

**Review by tosan88 — CHANGES_REQUESTED** [review]
Can you please also add a test case for this in `handler_test.go`?

File: app.go (hunk 1 above)
Comment (iulianp26) [reply app.go]: Typo fixed.

File: app.go (hunk 2 above)
Comment (iulianp26) [reply app.go]: Done.

**Review by tosan88 — APPROVED** [review]

File: circle.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+dependencies:
+  pre:
+    - go get github.com/axw/gocov/gocov; go get github.com/matm/gocov-html; go get -u github.com/jstemmer/go-junit-report
+test:
+  pre:
+    - go get github.com/mattn/goveralls
```
Comment (tosan88) [inline circle.yml:6]: Nice one!

**Review by kovacshuni — APPROVED** [review]

**Comment by kovacshuni** [comment]
great!

**Comment by kovacshuni** [comment]
🎄

**Pull request merged**